### PR TITLE
Spearman Dense Rank Correlation & Auto Connection to the Verticalab environment & Code cleaning

### PR DESCRIPTION
**Pull request by oualib**
- code cleaning
 - adding the method 'spearmanD' for vdf.acf, vdf.corr, vdf.corr_pvalue, vdf.hchart
 - New function verticalab_connection to automatically connect to the verticalab environment.
 - Removing drop_if_exists
 - Adding extra param raise_error to create_table
 - New function create_schema
 - Improving the .agg method to deal with incompatible aggregations.
 - SQL Magic: it is possible to use variables, option to export the results of queries to CSV/JSON, to run SQL files and to query vDataFrame or pandas.DataFrame or tablesamples.
 - new method .sort for tablesample
 - Splitting vDataFrame.count -> vDataFrame.count & vDataFrame.count_percent
 - new alias for regression and classification report -> report
 - Splitting vModel.predict -> vModel.predict & vModel.predict_proba. Normalising the code and making the functions more intuitive

### CODE REVIEW

**Review by ansleis — APPROVED** [review]

File: verticapy/learn/vmodel.py [hunk 1]
```diff
@@ -3755,24 +3837,104 @@ def predict(
...
         )
-        if len(self.classes_) == 2 and pos_label == None:
-            pos_label = self.classes_[1]
-        if inplace:
-            return vdf.eval(
-                name, self.deploySQL(pos_label=pos_label, cutoff=cutoff, X=X)
-            )
+
+    # ---#
+    def predict_proba(
```
Comment (afard) [inline verticapy/learn/vmodel.py:3868]: The same function is defined in line 3215!

File: verticapy/learn/vmodel.py [hunk 2]
```diff
@@ -3186,20 +3192,96 @@ def predict(
...
+        """
+        # Inititalization
+        if isinstance(X, str):
+            X = [X]
+        check_types(
+            [
+                ("name", name, [str]),
+                ("X", X, [list]),
+                ("vdf", vdf, [str, vDataFrame]),
+                ("inplace", inplace, [bool]),
```
Comment (afard) [inline verticapy/learn/vmodel.py:3257]: Don't you need check type for pos_label?

File: verticapy/learn/vmodel.py (hunk 1 above)
Comment (afard) [reply verticapy/learn/vmodel.py:3868]: Nevermind, I noticed that they belong to different classes.

**Review by afard — APPROVED** [review]

File: verticapy/learn/vmodel.py (hunk 2 above)
Comment (oualib) [reply verticapy/learn/vmodel.py:3257]: Done;

**Pull request merged**